assets/svg/magnifying-glass.svg

```diff
@@ -1 +1,3 @@
-<svg width="16" height="16" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M10.758 9.395 16 14.637 14.637 16l-5.242-5.242c-.979.705-2.159 1.135-3.449 1.135a5.946 5.946 0 1 1 5.947-5.947c0 1.29-.43 2.47-1.135 3.45ZM5.946 1.83A4.111 4.111 0 0 0 1.83 5.946a4.111 4.111 0 0 0 4.116 4.117 4.111 4.111 0 0 0 4.117-4.117A4.111 4.111 0 0 0 5.946 1.83Z" fill="currentColor"/></svg>
\ No newline at end of file
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M16 9.5C16 10.9069 15.5495 12.2145 14.7819 13.2827L20.49 19L19 20.49L13.2827 14.7819C12.2145 15.5495 10.9069 16 9.5 16C5.91 16 3 13.09 3 9.5C3 5.91 5.91 3 9.5 3C13.09 3 16 5.91 16 9.5ZM5 9.5C5 11.99 7.01 14 9.5 14C11.99 14 14 11.99 14 9.5C14 7.01 11.99 5 9.5 5C7.01 5 5 7.01 5 9.5Z" fill="#5F6368"/>
```

Comment:
fill should be set to `currentColor` and why the file was updated? 🤔

Reply:
The existing SVG icon was inappropriate size. Styling it didn’t work. Hence, I have exported a new one from the Figma design.